src/Microsoft.TestPlatform.Client/TestPlatform.cs

```diff
@@ -91,15 +90,15 @@ public TestPlatform()
         PopulateExtensions(discoveryCriteria.RunSettings, discoveryCriteria.Sources);
 
         // Initialize loggers.
-        var loggerManager = TestEngine.GetLoggerManager(requestData);
+        ITestLoggerManager loggerManager = TestEngine.GetLoggerManager(requestData);
```

Comment:
If we all agree, I could change the `var` rules in the repo to favor explicit type when not obvious.

My opinion: I usually prefer `var` in VS but that's true that `var` in GitHub review is a bit of a pain.

Reply:
I love var, but wgen it's not obvious it hard do discern that while reviewing. I would vote for yes.